pkg/sysched/sysched_test.go

```diff
@@ -0,0 +1,537 @@
+package sysched
+
+import (
+	//"fmt"
+	"bytes"
+	"context"
+	"encoding/json"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	informers "k8s.io/client-go/informers"
+	clientsetfake "k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/kubernetes/scheme"
+	restclient "k8s.io/client-go/rest"
+	restfake "k8s.io/client-go/rest/fake"
+	"k8s.io/kubernetes/pkg/scheduler/framework"
+	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/defaultbinder"
+	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/queuesort"
+	frameworkruntime "k8s.io/kubernetes/pkg/scheduler/framework/runtime"
+	st "k8s.io/kubernetes/pkg/scheduler/testing"
+	"net/http"
+	pluginconfig "sigs.k8s.io/scheduler-plugins/apis/config"
+	v1alpha1 "sigs.k8s.io/scheduler-plugins/pkg/sysched/clientset/v1alpha1"
+	"sigs.k8s.io/security-profiles-operator/api/seccompprofile/v1beta1"
+	"strings"
+	"testing"
```

Comment:
Please sort the imports by groups: (applies elsewhere)

- native imports (followed an empty line)
- upstream imports (followed by an empty line)
- imports in this repo